tests/src/test/java/io/streamnative/pulsar/handlers/kop/KafkaApisTest.java

```diff
@@ -498,7 +498,7 @@ public void testBrokerRespectsPartitionsOrderAndSizeLimits() throws Exception {
 
         int messagesPerPartition = 9;
         int maxResponseBytes = 800;
-        int maxPartitionBytes = 190;
+        int maxPartitionBytes = 900;
```

Comment:
Why did you do this change? Does `MessageFetchContext#readMessagesInternal` affect this test? This unit test is a known flaky test, I tried to change the `maxPartitionBytes` in my local environment but it still failed often.

Reply:
UT failed in https://github.com/streamnative/kop/runs/1345779596, and I didn't get the reason. I thought it may be entryNum * 190, and UT passed after a change to 900.

Reply:
It's a flaky test, the reason is still unknown. But it looks like not related to this param. So it's better to just keep the old value, I will rerun the CI if failed.